recipes/beem/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "beem" %}
+{% set version = "0.19.7" %}
+{% set sha256 = "8c71ee34cf17bda45e6db3fe6c619faa52e5fcb14342146b819bb22249e23110" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256|lower }}
```

Comment:
@justcalamari thoughts?

Reply:
Because our regex looks for `sha256:\s*[0-9A-Fa-f]+`, the `sha256: {{ sha256|lower }}` should not be replaced. It will find `{% set sha256 = "8c71ee34cf17bda45e6db3fe6c619faa52e5fcb14342146b819bb22249e23110" %}` and replace it with the new hash, so Jinja2 should render everything correctly. Note that https://github.com/conda-forge/biopython-feedstock/blob/master/recipe/meta.yaml does something very similar to this, and our PR into biopython-feedstock did not replace the Jinja2 variables. So this meta.yaml looks good to me.

Reply:
Awesome! Thank you!